### PR TITLE
chore: Release a new strings version 0.2.0

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Release a new [strings](https://github.com/objectionary/eo-strings) version 0.2.0

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@Graur Could you have a look, please?

File: objects/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -20,22 +20,76 @@
...
-
   s > @
 
+  [start len] > slice
+    text > @
+      ^.s.slice start len
+
   # Trim it from both sides
-  [] > trim /string
+  # @todo #107:30min Current implementation of trimmed method catches stackoverflow
```
Comment (Graur) [inline objects/org/eolang/txt/text.eo:40]: @volodya-lombrozo You forgot to remove todo here

File: objects/org/eolang/txt/text.eo [hunk 2]
```diff
@@ -99,19 +301,111 @@
...
+      minus.
+        length.
+          s
+        length.
+          other
 
-  # Returns an array of strings, separated by a given string
-  [delimiter] > split /array
+  # Returns an tuple of strings, separated by a given string
+  # @todo #22:30min Current implementation using Java.
```
Comment (Graur) [inline objects/org/eolang/txt/text.eo:358]: @volodya-lombrozo Same here

**Review by Graur — CHANGES_REQUESTED** [review]

**Comment by volodya-lombrozo** [comment]
@Graur Thank you. I've updated the PR and removed all todos. Could you take a look, again, please?

**Comment by Graur** [comment]
@volodya-lombrozo `make` check is pending. Maybe we can increase stack size [here](https://github.com/objectionary/home/blob/master/make/jvm/pom.xml) ? And check again

**Comment by volodya-lombrozo** [comment]
@Graur I've increased the stack-size, but seems that it didn't help.

**Comment by Graur** [comment]
@volodya-lombrozo What about your local environment? Did `make` script also fail?

**Comment by Graur** [comment]
@volodya-lombrozo Only this one is broken? Maybe we can disable it with `nop` object

**Comment by volodya-lombrozo** [comment]
@Graur I've tried to ignore ~2 tests and the problem remains. I believe it has to be solved somewhere either in other PR or in `eo-strings` repository. What do you think is the best way to fix it?

**Comment by volodya-lombrozo** [comment]
Was released here #116

**Pull request closed without merging**